builder.md

```diff
@@ -67,6 +67,54 @@ These files have the following structure:
       base0D: "dddddd"
       base0E: "eeeeee"
       base0F: "ffffff"
+    overrides:
+      default-background: base00
+      lighter-background: base01
+      selection-background: base02
+      comments: base03
+      invisibles: base03
+      line: base03
+      highlighting: base03
+      dark-foreground: base04
+      default-foreground: base05
+      caret: base05
+      delimiters: base05
+      operators: base05
+      light-foreground: base06
+      brightest-foreground: base07
+      variables: base08
+      xml-tags: base08
+      markup-link-text: base08
+      markup-lists: base08
+      diff-deleted: base08
+      integers: base09
+      boolean: base09
+      constants: base09
+      xml-attributes: base09
+      markup-link-url: base09
+      classes: base0A
+      markup-bold: base0A
+      search-text-background: base0A
+      strings: base0B
+      inherited-class: base0B
+      markup-code: base0B
+      diff-inserted: base0B
+      support: base0C
+      regular-expressions: base0C
+      regular-expressions: base0C
```

Comment:
Like the use of the overrides for more semantic naming, just a note the `regular-expressions` are duplicated here though